plugins/scaffolder/src/components/fields/EntityPicker/EntityPicker.tsx

```diff
@@ -61,14 +61,24 @@ export const EntityPicker = (props: EntityPickerProps) => {
   } = props;
   const catalogFilter = buildCatalogFilter(uiSchema);
   const defaultKind = uiSchema['ui:options']?.defaultKind;
+  const customFields = uiSchema['ui:options']?.fields;
   const defaultNamespace =
     uiSchema['ui:options']?.defaultNamespace || undefined;
 
   const catalogApi = useApi(catalogApiRef);
 
   const { value: entities, loading } = useAsync(async () => {
+    const defaultFields = [
+      'metadata.name',
+      'metadata.namespace',
+      'metadata.title',
+      'kind',
```

Comment:
`spec.profile` is useful here too and commonly has presentational data in it

Reply:
But like Ben says, the ideal solution would be if we only fetched kind, name and namespace here to be able to form entity refs, and then used the presentation API to decide what to draw.

That can be a follow-up though, imo, since that can get a bit more involved

Reply:
Yes, I want to add all attributes used as default in defaultEntityPresentation right now for the EntityPresentationApi integration ( kind, namespace, name, title, description, displayName, type )